app/factory/Layer.js

```diff
@@ -271,6 +271,26 @@ Ext.define('CpsiMapview.factory.Layer', {
 
         if (singleTile) {
 
+            if (layerConf.debounce !== undefined) {
+                var blankSrc = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkYAAAAAYAAjCB0C8AAAAASUVORK5CYII=';
+                var img;
+                var timeout;
+                olSourceConf.imageLoadFunction = function (image, src) {
+                    if (img) {
+                        img.src = blankSrc;
+                    }
+                    img = image.getImage();
```

Comment:
I find the names `img` and `image` a bit confusing, because on the first sight it is not clear to me what the difference is